driver/tke_driver.go

```diff
@@ -198,7 +200,12 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 	}
 	driverFlag.Options["zoneId"] = &types.Flag{
 		Type:  types.StringType,
-		Usage: "The zone id of the cluster",
+		Usage: "The zone id of the CVM Instance",
+	}
+	driverFlag.Options["imageId"] = &types.Flag{
+		Type:    types.StringType,
+		Usage:   "The image id of the cluster",
+		Default: &types.Default{DefaultString: "img-pyqx34y1"},
```

Comment:
What is a cluster image, is it the image for nodes?
Where does the default value come from? Does it depend on regions?

Reply:
It is the image for nodes. The imageId `img-pyqx34y1` corresponds to `ubuntu16.04.1 LTSx86_64`, it is the same as the default value of osName

Reply:
Is there a newer version available? e.g., 20.04